### PR TITLE
Created Scripts bundling and compilation configuration

### PR DESCRIPTION
**Pull request by william-abboud**
Added watching and transpiling of scripts in gulp

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions"]
```
Comment (william-abboud) [inline .babelrc]: So with this we can use ES6 features but if a browser already has caught up and implemented a feature we don't add code in

File: main.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import "babel-polyfill";
```
Comment (william-abboud) [inline main.js]: ES6 module syntax for the win + this imports all the amazing stuff that browser APIs haven't implemented yet

**Comment by william-abboud** [comment]
Will do in the next PR mate

**Comment by moonclash** [comment]
:shipit:

**Pull request merged**